armi/nuclearDataIO/tests/.gitignore

```diff
@@ -1,6 +1,3 @@
-*isotxs*
-*gamiso*
-*pmatrx*
```

Comment:
In retrospect this maybe didn't need to change. But also, it maybe wasn't necessary to have these in the .gitignore in the first place, so it's not clear that it's a problem.